omnigibson/controllers/joint_controller.py

```diff
@@ -56,13 +57,17 @@ def __init__(
                 joints that need to be processed in quaternion space to avoid gimbal lock issues normally faced by
                 3 DOF rotation joints. Each group needs to consist of three idxes corresponding to the indices in
                 the input space. This is only used in the delta_commands mode.
+            use_single_command (bool): whether all related joints should be controlled with one (scaled) input or
+                independent values individually. If True, then the inputted command should be a single value.
         """
         # Store arguments
         assert_valid_key(key=motor_type.lower(), valid_keys=ControlType.VALID_TYPES_STR, name="motor_type")
         self._motor_type = motor_type.lower()
         self._use_delta_commands = use_delta_commands
         self._compute_delta_in_quat_space = [] if compute_delta_in_quat_space is None else compute_delta_in_quat_space
 
+        self._use_single_command = use_single_command
```

Comment:
so how is this used exactly? doesn't there need to be additional logic to parse the single command and rescale / distribute it to all joints?

Reply:
Extend parallel gripper controller

Reply:
TODO: try to move to Parallel controller

Reply:
For now we'll use a jointcontroller still with N dimensions instead of 1, so that I can avoid having to implement things here. Putting it on the TODO list to improve the Gripper/Joint controller inheritance.

Reply:
As a result, we should get rid of `use_single_command` completely for now?